documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -1155,6 +1198,69 @@ The *MBean* is `debezium.oracle:type=connector-metrics,context=streaming,server=
 
 include::{partialsdir}/modules/all-connectors/ref-connector-monitoring-streaming-metrics.adoc[leveloffset=+1]
 
+[[oracle-monitoring-streaming-logminer]]
+==== Log Miner Metrics
+
+The *MBean* is `debezium.oracle:type=connector-metrics,context=log-miner,server=_<database.server.name>_`.
```

Comment:
This actually this should just be done as an Oracle-specific extension of the streaming metrics instead of being its own context (similar to what we do for MySQL). Could you log a follow-up Jira issue for this (it's down right now)?

Reply:
Logged as [DBZ-2537](https://issues.redhat.com/browse/DBZ-2537)